nix/packages/cardanow.nix

```diff
@@ -1,17 +1,18 @@
-{ writeShellApplication, cardanow-ts, cleanup-local-data, git, curl, cardano-cli, mithril-client-cli, jq, openssh, cardano-configurations, network ? "mainnet" }:
+{ writeShellApplication, cardanow-ts, cleanup-local-data, git, curl, cardano-cli, mithril-client-cli, jq, openssh, cardano-configurations, postgresql_14, network ? "mainnet" }:
 writeShellApplication
 {
   name = "cardanow";
   runtimeEnv.NETWORK = network;
   runtimeInputs = [
+    cardano-cli
     cardanow-ts
     cleanup-local-data
     curl
     git
-    cardano-cli
-    mithril-client-cli
     jq
+    mithril-client-cli
     openssh
+    postgresql_14
```

Comment:
This has to corresponds with the postgres service running in docker, right? Do we want to add a comment perhaps?